srv/components/csm/files/ha-params.tmpl

```diff
@@ -5,5 +5,8 @@
 {%- endif -%}
 interface: {{ mgmt_if }}
 vip: {{ pillar['cluster']['mgmt_vip'] }}
+{% set netmask = pillar['cluster'][grains['id']]['network']['mgmt_nw']['netmask'] %}
+{% set cidr = salt.network.calc_net(pillar['cluster']['mgmt_vip'], netmask) %}
+cidr: {{ cidr.split("/")[1] }}
```

Comment:
It would be better not to perform any operations here, unless necessary

Reply:
Yes, sure. Updated code for proper template rendering